applications/doodle/src/doodle_util.erl

```diff
@@ -155,6 +156,7 @@ save_sms(JObj, DocId, Doc, Call) ->
                ,{<<"request_user">>, RequestUser }
                ,{<<"request_realm">>, RequestRealm }
                ,{<<"body">>, Body }
+               ,{<<"length">>, Length }
```

Comment:
Would you be opposed to naming it "bit_length" or "bits", since typically "length" implies bytes?